src/components/Calculator.js

```diff
@@ -0,0 +1,19 @@
+/*eslint-disable */
```

Comment:
- [ ] It's understandable if you have one or two eslint rules that you don't know how to fix. But disable all eslint I think is not appropriate, I invite you to quit it and fix all issues that you could have. Eslint is important because this helps you to get a better and more professional code, at the beginning could be hard to follow it, but you'll see how you improve your code following this.
- Hint: If you install everything for [linters config locally](https://github.com/microverseinc/linters-config/tree/master/react-redux) you could use `npx eslint .` to see the mistakes and then use `npx eslint . --fix` to fix the most. Please, consider this, is the easiest way to do it.
![image](https://user-images.githubusercontent.com/20563252/130177581-3d28919d-6347-4c17-839b-6cca04579a52.png)

Reply:
- Hello again, talking specifically to the `react/prefer-stateless-function` it's ok if you create an exception on the script, but for the others, please, work on it. Sorry if I don't say this before. 🐱

Reply:
> ok working on it

Reply:
- corrected

Reply:
> corrected